core/vm/instructions.go

```diff
@@ -37,45 +37,45 @@ var (
 	errMaxCodeSizeExceeded   = errors.New("evm: max code size exceeded")
 )
 
-func opAdd(pc *uint64, evm *EVM, contract *Contract, memory *Memory, stack *Stack) ([]byte, error) {
+func opAdd(pc *uint64, interpreter *InterpreterEVM, contract *Contract, memory *Memory, stack *Stack) ([]byte, error) {
```

Comment:
Imho rename `interpreter` to `evm` throughout this file. Much cleaner and shorter code.

Reply:
I have an issue with this, because it's an `EVMInterpreter` and not an `EVM`. It is shorter but confusing. What about `evmi` ?